activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+*   `delegate` now defines method with proper arity when delegating to a class.
+    With this change, it defines faster method (3.5 times faster with no argument),
+    but in this case, the delegation target method has to be defined before
+    declaring the delegation.
+
+    ```ruby
+    # This works, and defines 3.5 times faster method than before
+    class C
+      def self.x() end
+      delegate :x, to: :class
+    end
+
+    class C
+      # This doesn't work because `delegate` cannot find the definition of `x`
```

Comment:
If I understood correctly, this used to work, but it doesn't anymore. Is that correct?

If so, should we note that this is a breaking change?